test/hotspot/jtreg/testlibrary/ctw/Makefile

```diff
@@ -42,11 +42,12 @@ JAVAC = $(JDK_HOME)/bin/javac
 JAR = $(JDK_HOME)/bin/jar
 
 SRC_FILES = $(shell find $(SRC_DIR) -name '*.java')
-LIB_FILES = $(shell find $(TESTLIBRARY_DIR)/jdk/test/lib/ \
+# Exclude ModuleInfoWriter.java to circumvent '--enable-preview'.
```

Comment:
Wording: `Exclude files that need --enable-preview to compile`. There would probably be more files later.

Reply:
I took a look at LIB_FILES. Only 'ModuleInfoWriter.java' depends on advanced APIs. 
It was added to testlibrary in [JDK-8304163](https://bugs.openjdk.org/browse/JDK-8304163).

Yes, we may need to exclude more files in the future. Currently, Makefile selects LIB_FILES using wildcard matching. If it's necessary, we need to define LIB_FILES explicitly.